docs/users/view-geyser.md

```diff
@@ -102,10 +102,7 @@ You can view the latest information on a geyser, including the latest eruptions
   </TabItem>
   <TabItem value="iOS" label="iOS">
 
-    * Open the GeyserTimes app.
-    * Tap the geyser you want to see on a map. 
-    * In the tabs, tap **Overview**.
-    * Tap the **Pushpin icon (📍)**.
+    This feature is not available on the app for iOS yet.
```

Comment:
Please revert, as this feature will be available once the help center is published.